File: pgm/ABC_R/ABC_358_A.r
con <- file(description = "stdin", open = "r")
ST <- readLines(con = con, n = 1) |>
    strsplit(split = " ") |>
    unlist()
close(con)
Yes <- "Yes"
No <- "No"
AtCoder <- "AtCoder"
Land <- "Land"
S <- ST[1]
t <- ST[2]
ifelse(S == AtCoder && t == Land, Yes, No) |>
    cat("\n", sep = "")

File: pgm/ABC_Dart/ABC_358_A.dart
// import 'dart:collection';
import 'dart:io';
// import 'dart:math';
// import 'dart:typed_data';

void main() {
  final [String S, String T] = stdin.readLineSync()!.split(' ');
  const String AtCoder = 'AtCoder';
  const String Land = 'Land';
  const String Yes = 'Yes';
  const String No = 'No';
  if (S == AtCoder && T == Land) {
    print(Yes);
  } else {
    print(No);
  }
  exit(0);
}
